code/src/store/api/wishlistApi.ts

```
@@ -0,0 +1,45 @@
+import { rtkQueryTags } from "@/constants/api-tags";
+
+import {
+  GetUserProductsParams,
+  GetUserProductsResponse,
+} from "@/types/product.types";
+
+export const wishlistApi = appApi.injectEndpoints({
+  endpoints: (build) => ({
+    getUserWishlist: build.query<
+      GetUserProductsResponse,
+      GetUserProductsParams | void
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)